.github/workflows/core-docker-tag.yml

```diff
@@ -0,0 +1,29 @@
+name: Kinto Core Docker Image Tag
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
do we need this "uses" line?

Reply:
Same comment, no need to checkout code here. we are only tagging existing image so line 16 and 17 can go away too.